src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -84,14 +88,16 @@ const makeId = makeGenerator()
  * "longitude": -122.45,
  * "zoom": 15,
  * "controller": true,
+ * "showingLabels": true, // Enables presenting labels when hovering over Geo_Point.
  * "layers": [{
  *     "type": "Scatterplot_Layer",
  *     "data": [{
  *         "type": "Geo_Point",
  *         "latitude": -122.45,
  *         "longitude": 37.8,
  *         "color": [255, 0, 0],
- *         "radius": 100
+ *         "radius": 100,
+ *         "label": "foo bar baz"
```

Comment:
use "an example label" instead of "foo bar baz" pls. It looks not very professional IMO :)